types/detox/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
This file should only `"extends": "dtslint/dt.json"` for new packages.

Reply:
Do you mean it shouldn't have any rules?

Reply:
Yes, just `{"extends": "dtslint/dt.json"}`.